style.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-gap: 10px;
+  background-color: #e94a4a;
+  padding: 10px;
+}
+
+
+
+
+
```

Comment:
- [ ] Please be more, clear, and polite in your comments and try to tell why students should follow your advice like spaces in your code makes code lengthier and difficult to maintain even if it can lead any person in the wrong direction because of confusion.

### Your Comment

> Amazing templates made for your code!. Kindly, could you eliminate blank spaces?

Reply:
Your code is amazing. You are too short to be a Master! 💯 

I suggest reading more information about how to write code clearly. 🐰 Sometimes we forgot some things.

Please, delete all blank spaces that are not necessary.

🍓 💪 🍓